### PR TITLE
Converter refactoring and test

### PR DESCRIPTION
**Pull request by grzkv**
1. Added basic unit tests for the `Converter` class;
2. Added `toString` conversion to several `Cell` subclasses. The conversion was absent and `Object.toString()` was called. This caused unit tests to fail.

### CODE REVIEW

File: src/test/java/com/booking/replication/schema/column/types/ConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    // TODO : Add tests for overflows
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    // REAL NUMBERS
+
+    @Test
+    public void doubleCell() throws TableMapException {
+        Cell c = new DoubleCell(1.5);
+        ColumnSchema s = new ColumnSchema();
+
```
Comment (bdevetak) [inline src/test/java/com/booking/replication/schema/column/types/ConverterTest.java]: Here the setting of the column type and data type for the  ColumnSchema object is missing.

File: src/test/java/com/booking/replication/schema/column/types/ConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    public void floatCell() throws TableMapException {
+        Cell c = new FloatCell((float)1.5);
+        ColumnSchema s = new ColumnSchema();
+        s.setColumnType("float");
+
+        assertEquals("1.5", Converter.cellValueToString(c, s));
+    }
+    
+    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    // TIME AND DATE
```
Comment (dvbobrov) [inline src/test/java/com/booking/replication/schema/column/types/ConverterTest.java]: TODO: test with timezones, tests for incorrect dates

File: src/test/java/com/booking/replication/schema/column/types/ConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    // STRING CELLS
+
+    @Test
+    public void stringToString() throws TableMapException {
+        String testString = "test";
+        Cell stringCell = StringCell.valueOf(testString.getBytes(StandardCharsets.UTF_8));
+        ColumnSchema stringSchema = new ColumnSchema();
+        stringSchema.setCharacterSetName("utf8");
```
Comment (dvbobrov) [inline src/test/java/com/booking/replication/schema/column/types/ConverterTest.java]: TODO: different charsets

**Comment by grzkv** [comment]
Thanks @bdevetak @dvbobrov

**Pull request merged**